package.json

```diff
@@ -36,7 +36,9 @@
   },
   "scripts": {
     "start": "NODE_ENV=development webpack --progress --watch --display-error-details --display-modules --display-reasons --mode development",
-    "build": "composer dump-autoload && NODE_ENV=production webpack -p --progress --mode production"
+    "build": "composer dump-autoload && NODE_ENV=production webpack -p --progress --mode production && npm run script:licenses",
+    "precommit": "eslint assets/dev/scripts/*.js && stylelint \"assets/dev/styles/**/*.scss\" --syntax scss && ./vendor/bin/phpcs --ignore=*/vendor/*,*/node_modules/* .",
```

Comment:
The `precommit` script can be removed, since it's added in the `husky` hooks 🙂

Reply:
oh yeah I added it back in so i could do `yarn run precommit` instead of actually committing and hoping.   :P - is there a way to run the husky scripts direct other than the git triggers?

Reply:
You can check the code using composer scripts 🙂 Just do `composer check-cs .` and it should work. The `eslint` and `stylelint` could be added in their own scripts in the `package.json` so that it could be run independently. 

I mean we can leave it here, I just wonder if that could interfere with the husky script.